### PR TITLE
fix brick01 file name

### PR DESCRIPTION
**Pull request by greyltc**

### CODE REVIEW

**Comment by vocx-fc** [comment]
LOL. I've seen this error for the longest time but never got the time to correct it. Now I was thinking of submitting the patch but other things got in the way. I'm glad that it's finally going to be fixed.

**Comment by greyltc** [comment]
I see a red X here. Somehow I doubt that's the fault of this PR. Is there anything more I should do now to get this merged?

**Comment by luzpaz** [comment]
Please force push the commit instead of merging. It will overwrite the original commit instead of committing over it. In other words, with the former method you end up with 1 commit, the latter with 2 commits (this is what you did). 

See https://stackoverflow.com/a/12610763

**Comment by greyltc** [comment]
> Please force push the commit instead of merging.

Hm. I clicked the "Update branch" button I saw in GitHub's friendly UI:
image
Let me see how I can undo that...

**Comment by greyltc** [comment]
Alright @luzpaz, I've blown away the merge commit that the GitHub button gave me. I think this PR should be back to only one commit now.

**Comment by vocx-fc** [comment]
Don't worry too much about what luzpaz says. Pressing the "update branch" button is fine. What it does is basically rebase your branch against the latest master; a second commit won't be recorded. Updating the branch in this way will basically test your branch against the latest master and tell you if there are any conflicts. Since your change is extremely small there is almost no chance for a conflict so pressing the button (rebasing) is not even necessary.

The red X that you see indicates that the automated tests that the system runs when merging your commit failed. If you click on the details it will take you to the Travis page which will give you more information. Currently all commits are failing the tests due to an unrelated error, so don't worry, your commit is fine.

Only administrators can merge your commit, even if the tests fail. Since it's Christmas and New Year don't expect this to be merged immediately. Let the admins have some punch and it will be merged later.

**Comment by luzpaz** [comment]
> Don't worry too much about what luzpaz says. Pressing the "update branch" button is fine. What it does is basically rebase your branch against the latest master; a second commit won't be recorded. 

I disagree, I think the 2nd commit _does_ get recorded. Furthermore if one of the devs that merges commits doesn't squash the PR it will be added to the commit history, which is something we're trying to keep as organized as possible.

**Comment by vocx-fc** [comment]
The merge is recorded in the commit history if the user is working in his own branch offline, and he or she manually merges the master branch. This is done a lot by the Path guys, for example, #2784 (a00eec92), #2801 (b3c4ef89).

But this is a problem of them. They shouldn't do it like that. They should rebase their own branch on top of the online master, and then push online. Instead, it seems that they continually merge the master instead of rebasing. The way they do it is odd and generates additional commits.

If you make a pull request then pressing the "Update branch" button won't generate a new commit. It will just rebase with the current master, and allow you to see if there are any conflicts with the current code.

See for example my #2727, which was merged in fa73d29d. The pull request has two such "Merge branch 'master'...", but there is only the single fa73d29d commit. No additional commits are recorded after this one. See the [commit history of the next commit](https://github.com/FreeCAD/FreeCAD/commits/92fa3815). The next commit corresponds to another pull request, #2770. The two "Merge branch 'master'..." from the first pull request were not recorded.

As I said, it rebases with the current code, to see if there are conflicts. If there was a failed test in Travis, you can press the "Update branch" to run again the Travis tests. This is good if you want to show that your pull request really passes the tests, as often the fail in Travis is not caused by your branch but by another branch that was merged before yours.

In summary, don't be afraid of pressing the "Update branch" button. It is fine. The extra commits will be added only if you are sloppy managing your code offline before pushing it online.

**Comment by luzpaz** [comment]
@vocx-fc Interesting. Thanks for your thorough explanation. It makes sense then to let the Path folks (CC @sliptonic @mlampert) know about this :point_up_2:
